src/test/java/no/sikt/nva/pubchannels/handler/CreateJournalHandlerTest.java

```diff
@@ -54,6 +57,7 @@ class CreateJournalHandlerTest {
     public static final String HEADER_AUTHORIZATION = "Authorization";
```

Comment:
Seems like most of these constants are only used internally in this class. Make them private.